README_TTN.md

```diff
@@ -103,6 +107,139 @@ Details are available [here](https://blog.squix.org/2017/07/seeeduino-lora-gps-g
+
+The code below is the comand, already configured, that we use in order to connect.
+
+Two keys, one for each application are generated in order to
+connect and used as password.
+Remember to remove the security code.
+
+```bash
+curl --location --request POST 'localhost:4042/iot/services' \
+--header 'fiware-service: openiot' \
+--header 'fiware-servicePath: /' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "services": [
+    {
+      "entity_type": "WasteContainer",
+      "apikey": "",
+      "resource": "8CF957200005727C",
+      "attributes": [
+        {
+          "object_id": "temperature_1",
+          "name": "temperature",
+          "type": "Number"
+        },
+        {
+          "object_id": "relative_humidity_1",
+          "name": "relativeHumidity",
+          "type": "Number"
+          },
+          {
+            "object_id": "digital_in_1",
+            "name": "flameDetected",
+            "type": "Number"
+            },
+            {
+              "object_id": "digital_in_2",
+              "name": "fillingLevelPercentage",
+              "type": "Number"
+            },
+            {
+              "object_id": "accelerometer_1",
+              "name": "accelerometerXYZ",
+              "type": "StructuredValue"
+            },
+            {
+              "object_id": "accelerometer_2",
+              "name": "gAccellerationXYZ",
+              "type": "StructuredValue"
+            },
+            {
+              "object_id": "digital_in_3",
+              "name": "status",
+              "type": "Number"
+            }
+          ],
+          "internal_attributes": {
+            "lorawan": {
+              "application_server": {
+                "host": "eu1.cloud.thethings.network",
+                "username": "oc-diorama-001@ttn",
+                "password": "NNSXS.KQRRO4R7BXMHELHQZOMELYUJ2GQUU4S5EI2I6QY.P7FJVWQRODOLCOHOAC2GQMEDO65LINW6YAOBKL6YMUU2K5C4R5MAC0MM3NT01T4L1ANO",
+                "provider": "TTN"
+                },
+                "app_eui": "8CF957200005727C",
+                "application_id": "oc-diorama-001@ttn",
+                "application_key": "51F58CAC3F5735E3D1F88DD3EADBE9C6",
+                "data_model": "application_server"
+            }
+          }
+    },
+    {
+      "entity_type": "environment",
+      "apikey": "",
+      "resource": "8CF95720000569A6",
+      "attributes": [
+        {
+          "object_id": "digital_in_1",
+           "name": "NO2",
+           "type": "Number"
+        },
+        {
+          "object_id": "digital_in_2",
+          "name": "C2H50H",
+          "type": "Number"
+        },
+        {
+          "object_id": "digital_in_3",
+          "name": "VOC",
+          "type": "Number"
+        },
+        {
+          "object_id": "digital_in_4",
+          "name": "CO",
+          "type": "Number"
+        },
+        {
+          "object_id": "digital_in_5",
+          "name": "airQuality",
```

Comment:
airQualityLevel

let's use numbers from international standards to compute the level:

https://www.airnow.gov/aqi/aqi-basics/

i.e. for no pollution, a number below 50 etc (to be fixed in arduino code)

Reply:
i don't think this is solved looking at the code .ino

Reply:
ok, it's solved in the expression